gflownet.py

```diff
@@ -533,10 +555,10 @@ def sample_queries(self, nb_queries):
         return batch
 
     def manip2policy(self, state):
-        seq_manip = state
+        seq_manip = np.array(state)
```

Comment:
To be moved !! Either in env. Either in preprocess in the GFN model

Reply:
Yeah but that would be a separate PR. Have raised an issue for that